2020/06.md

```diff
@@ -81,7 +81,8 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 3     | 15m     | Changing Promise.any's AggregateError's argument processing order ([PR](https://github.com/tc39/proposal-promise-any/pull/59)) | Shu-yu Guo |
     |   | 2     | 60m     | [Function Implementation Hiding](https://github.com/tc39/proposal-function-implementation-hiding) for stage 3 ([slides](https://docs.google.com/presentation/d/1zCACnOkueOZHruXv2UzUvKIUA6L7w7W6S2N4bdzwwkM/edit)) ([spec text](https://github.com/tc39/ecma262/pull/1739)) | Michael Ficarra |
     |   | 2     | 60m     | [Realms, Stage 2 updates](https://github.com/tc39/proposal-realms) (slides TBD) | Caridy Patiño, Leo Balter |
-    |   | 1     | 30m     | [Ergonomic brand checks for private fields][private-fields-in-in] for stage 2 | Jordan Harband
+    |   | 1     | 30m     | [Ergonomic brand checks for private fields][private-fields-in-in] for stage 2 | Jordan Harband |
+    |   | 0     | 30m     | [Intl Duration Format](https://github.com/tc39/proposal-intl-duration-format) for Stage 1 (slides WIP) | Younies Mahmoud |
```

Comment:
I'd suggest linking to your slides, even if they're not fully finished.

Reply:
Done!